added back to repo (#4)
WayScience/CytoCherryPick: README.md

> # CytoCherryPick

Reviewer: Consider adding a description for where the project name comes from (for example, a phrase definition, or cultural reference).

Author: Updated!

## Welcome to CytoCherryPick
CytoCherryPick is an open source software that provides a set of tools for the selection of representative images from a set of images.

> ## Functions of CytoCherryPick

Reviewer: Very optional consideration: the use of the word "features" here gets overloaded due to the focus area ("image features" etc), but generally in software one can label "useful functionality / functions" as "features". For example, verbally, "CytoCherryPick provides the following software features to the audience...". Doing this also decouples you from the need to distinguish between whether you're providing functions vs classes vs other software components (it might get abstract along the way 🙂 ).
```suggestion
## Features of CytoCherryPick
```

* **Image Selection**: CytoCherryPick will select a representative image from a set of images.
* **Image Distribution Analysis**: Choose your own representative image and CytoCherryPick will graph where in the image set distribution your image is located.

> ### Install the CytoCherryPick to your environment

Reviewer: Very optional: consider shortening this header. Sometimes the way people install Python work may not be into what they think of as an "environment". That said, simplification can help make the documentation here more approachable.
```suggestion
### Installation
```

# CytoCherryPick

## Welcome to CytoCherryPick

> CytoCherryPick is an opensource sowftware that provides a set of tools for the selection of representative images from a set of images.

Reviewer: ```suggestion
CytoCherryPick is an open source software that provides a set of tools for the selection of representative images from a set of images.
```

## Functions of CytoCherryPick
* **Image selection**: CytoCherryPick will select a representative image from a set of images.

> * **Image Distribution Analysis**: Choose your own representative image and CytoCherryPick will graph where in the image set distribution your image is located.

Reviewer: Consider maintaining consistent capitalization for the bolded text at the beginning of each line in this list.

# Installing and Using CytoCherryPick

### Copy the repository
``` bash
$ git clone
$ cd CytoCherryPick
```

> ### Install the CytoCherryPick env

Reviewer: Consider completing the abbreviated `env` here as `environment` or defining it as an abbreviation somewhere before this line.

``` bash

> $ mamba env install --file=env/env.yml

Author: hmmm, I am not sure. This realm is new to me and we might need to talk more about how I can do this. I will be picking your brain later!

# CytoCherryPick
Cyto: A cell.

> CherryPick: To select the best or most desirable.

Reviewer: Did these definitions come from a certain resource? Consider referencing one if so.
